packages/ng/CONTRIBUTING.md

```diff
@@ -0,0 +1,7 @@
+ - **DO** use english
+ - **DO** append the changelog when you fix an issue or add a feature
+ - **DO** reference the issue you're fixing in the PR and changelog
+ - **DO** flag your PR with the right labels, especially `ng` and `BREAKING CHANGE`
+ - **DO** launch `ng test -a=src` and `ng lint -a=src` before submitting, if it fails your PR wont be approved

```

Comment:
It's better to use `npm test` and `npm run lint` in order to use the same command in all packages